impl/src/test/groovy/io/jsonwebtoken/JwtsTest.groovy

```diff
@@ -171,33 +173,13 @@ class JwtsTest {
         }
     }
 
-    @Test
-    void testParseWithHeaderOnly() {
```

Comment:
We need to keep this test, since this amounts to an `Unsecured JWT` and they are supported.  If anything, instead of using `foo..` as the test string, we should use:

```groovy
Encoders.BASE64URL.encode('{"alg":"none"}') + '..'
```